src/App.css

```diff
@@ -101,22 +101,78 @@ nav {
   width: 60%;
 }
 
-#comments{
+#comments {
   margin-top: 20px;
   padding-bottom: 20px;
-  width:100%;
+  width: 100%;
 }
-#commentsTitle{
+#commentsTitle {
   padding-top: 10px;
   padding-bottom: 10px;
 }
-#commentsList{
-  list-style:none;
+#commentsList {
+  list-style: none;
 }
-.commentsListItem{
+.commentsListItem {
   margin-bottom: 10px;
-  border:solid 2px rgb(26, 102, 96);
+  border: solid 2px rgb(26, 102, 96);
   background-color: rgb(42, 126, 119);
   padding-top: 5px;
   padding-bottom: 5px;
 }
+
+.upvoteButton-1 {
```

Comment:
In regards to your styling in general, there are quite a few low contrast mixes on your page e.g. white text on the voting buttons are impossible to read on the light blue background. Please change these